run-tests.sh

```diff
@@ -97,19 +97,19 @@ exit_error() {
 }
 
 test_service() {
-	mvn clean verify -B -Pservice_testing -Djdbc.url=$JDBC_URL -Dirida.it.rootdirectory=$TMP_DIRECTORY -Dsequence.file.base.directory=$SEQUENCE_FILE_DIR -Dreference.file.base.directory=$REFERENCE_FILE_DIR -Doutput.file.base.directory=$OUTPUT_FILE_DIR -Djdbc.pool.maxWait=$DB_MAX_WAIT_MILLIS $@
+	mvn clean verify -B -Pservice_testing -Djdbc.url=$JDBC_URL -Dfile.processing.decompress=true -Dirida.it.rootdirectory=$TMP_DIRECTORY -Dsequence.file.base.directory=$SEQUENCE_FILE_DIR -Dreference.file.base.directory=$REFERENCE_FILE_DIR -Doutput.file.base.directory=$OUTPUT_FILE_DIR -Djdbc.pool.maxWait=$DB_MAX_WAIT_MILLIS $@
```

Comment:
Is the default `file.processing.decompress=true`?  If so does this need set for all these tests?

Reply:
Yes, that is the default. The reason I'm setting it explicitly here is if you have set `file..processing.decompress=false` in your `/etc/irida/irida.conf` file, and then run the tests on your own machine (like I sometimes do), they won't work properly.

Reply:
Ah understood.  Yeah that's good thanks.